app/controllers/application_controller.rb

```diff
@@ -1,2 +1,13 @@
 class ApplicationController < ActionController::Base
+  before_action :set_locale
+
+  def set_locale
+    locale = params[:locale].to_s.strip.to_sym
+
+    I18n.locale = if I18n.available_locales.include?(locale)
+                    locale
+                  else
+                    I18n.default_locale
+                  end
```

Comment:
```suggestion
    I18n.locale = I18n.available_locales.include?(locale) ? locale : I18n.default_locale
```
Viết như này được không em